src/Native/Runtime/portable.cpp

```diff
@@ -35,7 +35,7 @@
 #include "GCMemoryHelpers.h"
 #include "GCMemoryHelpers.inl"
 
-#if defined(USE_PORTABLE_HELPERS) || defined(_ARM_)
+#if defined(USE_PORTABLE_HELPERS)
```

Comment:
(It should be basically a rollback of https://github.com/dotnet/corert/pull/3989/)